cdk/app.py

```diff
@@ -1,11 +1,20 @@
 #!/usr/bin/env python3
+import boto3
 from aws_cdk import core
 from ruamel.yaml import SafeLoader
 from ruamel.yaml import load as yaml_load
 
 from domino_cdk.config import config_loader
 from domino_cdk.domino_stack import DominoStack
 
+sts = boto3.client("sts")
+try:
+    sts.get_caller_identity()
```

Comment:
I have no idea if this is true, but in theory could we have credentials that work without STS access?

Reply:
https://docs.aws.amazon.com/STS/latest/APIReference/API_GetCallerIdentity.html

```
No permissions are required to perform this operation. If an administrator adds a policy to your IAM user or role that explicitly denies access to the sts:GetCallerIdentity action, you can still perform this operation. Permissions are not required because the same information is returned when an IAM user or role is denied access. To view an example response, see I Am Not Authorized to Perform: iam:DeleteVirtualMFADevice in the IAM User Guide.
```

Reply:
Doesn't seem like this is a concern.